app/views/works/show.html.erb

```diff
@@ -0,0 +1,38 @@
+<%# find me at http://localhost:3000/works/2 %>
+<main>
+  <section>
+<h1><%= @work.title %></h1>
+<p>Created By: <%= @work.creator %></p>
+<p>Published: <%= @work.publication_date %></p>
+
+<p><%= @work.description%></p>
+<p><%= link_to "Back to Media Ranks", works_path %></p>
+<p><%= link_to "Edit", works_path %></p>
```

Comment:
this is the wrong path. it should be `edit_works_path`. You can get that information using `rails routes` in the terminal